Makefile

```diff
@@ -48,8 +35,6 @@ coverage:
 	xcrun llvm-cov export -format="lcov" .build/debug/mongo-swift-driverPackageTests.xctest/Contents/MacOS/mongo-swift-driverPackageTests -instr-profile .build/debug/codecov/default.profdata > info.lcov
 
 clean:
-	rm -rf Packages
 	rm -rf .build
-	rm -rf MongoSwift.xcodeproj
```

Comment:
this is doubly irrelevant because 1) the generated xcodeproj is actually named `mongo-swift-driver.xcodeproj` ever since we renamed the package, and now also 2) no more `.xcodeproj` needed to build